### PR TITLE
Feature: team members block

### PR DESCRIPTION
**Pull request by josadcha**

### CODE REVIEW

File: src/components/Team/index.js [hunk 1]
```diff
@@ -29,6 +30,20 @@ const TEAM_HEADLINERS = [
 ];
```
Comment (kh-ruslan) [inline src/components/Team/index.js:12]: You can simplify the codebase by defining the `gradient` outside:
```
const gradient = 'linear-gradient(0deg, #090909 0%, rgba(9, 9, 9, 0) 100%)';
```
and then using it in `style` function as in suggestion below

File: src/components/Team/index.js [hunk 2]
```diff
@@ -29,6 +30,20 @@ const TEAM_HEADLINERS = [
...
 
 export default function Team() {
+    const [hoverId, setHoverId] = useState(-1);
+
+    const style = (id, image) => {
```
Comment (kh-ruslan) [inline src/components/Team/index.js]:
```suggestion
    const buildStyle = (id, image) => {
        const isActive = hoverId === id;

        return {
            backgroundImage: `${isActive ? `${gradient}, ` : ''}url(${useBaseUrl(`/img/photos/${image || 'anonymous.png'}`)})`,
        }
    };
```

**Comment by kh-ruslan** [comment]
Team block creates an extra space on the right side on resolution >966 and <1440:
image

**Comment by josadcha** [comment]
LGTM

**Pull request merged**